src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -555,6 +564,47 @@ end = struct
     ; user_commands_count= List.length user_commands
     ; coinbase_parts_count= List.length coinbase }
 
+  let check_zero_fee_excess scan_state data =
```

Comment:
since there's no .mli file, should the function have a brief description in a doccomment?

Reply:
This function is not exposed (in `Coda_pow`). It is a helper function for `apply_diff`. Also, I will be writing documentation for staged ledger as a whole. So some of these checks would be listed in there.

Reply:
May as well comment explaining what the function does so you don't forget before adding more comments later

Reply:
You mean in `Staged_ledger.ml` itself right?

Reply:
yeah I mean here